src/content/developers/docs/scaling/optimistic-rollups/index.md

```diff
@@ -1,46 +1,254 @@
+2. If a basic rollup transaction uses 12 bytes, then the average Ethereum block can process **78,125 rollup transactions** (937,5000/12) or **39 rollup batches** (if each batch holds an average of 2000 transactions).
+3. If a new block is produced on Ethereum every 15 seconds, then the rollup's processing speeds would amount to roughly **5,208 transactions per second**. This is done by dividing the number of basic rollup transactions an Ethereum block can hold (78,125) by the average block time (15 seconds). 
+
+This is a fairly optimistic estimate, given that optimistic rollup transactions cannot possibly comprise an entire block on Ethereum. However, it can give give a rough idea of how much scalability gains that optimistic rollups can afford Ethereum users (current implementations offer up to 2,000 TPS). 
+
+The introduction of [data sharding](/upgrades/shard-chains/) on Ethereum is expected to improve scalability in optimistic rollups. Because rollup transactions must share blockspace with other non-rollup transactions, their processing capacity is limited by data throughput on the main Ethereum chain. Sharding will increase the space available to L2 chains to publish data per block, further boosting throughput on rollups. 
+
+### Pros and cons of optimistic rollups {#optimistic-rollups-pros-and-cons}
 
 | Pros                                                                                                             | Cons                                                                        |
 | ---------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
-| Anything you can do on Ethereum layer 1, you can do with Optimistic rollups as it's EVM and Solidity compatible. | Long wait times for on-chain transaction due to potential fraud challenges. |
-| All transaction data is stored on the layer 1 chain, meaning it's secure and decentralized.                      | An operator can influence transaction ordering.                             |
+|Offers massive improvements in scalability without sacrificing security or trustlessness.    | Delays in transaction finality due to potential fraud challenges.                                                                                                     |
+| Transaction data is stored on the layer 1 chain, improving transparency, security, censorship-resistance, and decentralization. | Centralized rollup operators (sequencers) can influence transaction ordering.                                                                   | 
+| Fraud proving guarantees trustless finality and allows honest minorities to secure the chain.                     | A malicious operator can steal funds by posting invalid blocks and state commitments.                                                                              | 
```

Comment:
If there are no honest nodes a malicious operator can...

It could happen, but it's extremely unlikely.

Reply:
Sure, but I think it was still important to point out. If you notice, I mentioned in the next line: "Security model relies on at least one honest node executing rollup transactions and submitting fraud proofs to challenge invalid state transitions."

Reply:
I agree it needs to be pointed out. I'm just suggesting that 

> If there are no honest nodes a malicious operator can steal funds by posting invalid blocks and state commitments

is more accurate than 

> A malicious operator can steal funds by posting invalid blocks and state commitments

The text here makes it seem like it's unavoidable, at least until they read the next line.

Reply:
That makes sense. I just changed the text to what you suggested.